extensions/wikia/Search/classes/Config.php

```diff
@@ -1347,13 +1566,16 @@ public function setXwikiArticleThreshold( $xwikiArticleThreshold ) {
 
 	/**
 	 * Sanitizes limit so it is numeric, positive and smaller than the maximum
+	 *
 	 * @param $limit
+	 *
 	 * @return int
 	 */
 	protected function sanitizeLimit( $limit ) {
 		$limit = intval( $limit );
 		$limit = $limit > 0 ? $limit : self::RESULTS_PER_PAGE;
 		$limit = $limit < self::MAX_RESULTS_PER_PAGE ? $limit : self::MAX_RESULTS_PER_PAGE;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$limit". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)